deploy/concierge/values.yaml

```diff
@@ -47,7 +47,7 @@ custom_labels: { }
 
 #@schema/title "Replicas"
 #@schema/desc "Specify how many replicas of the Pinniped server to run."
-replicas: 2
+replicas: 1
```

Comment:
We need to remember not to merge this change.

Reply:
Good catch, I did this just to help manage the log volume

Reply:
That's what I figured. I've reverted it now.